### PR TITLE
updates to website

### PR DESCRIPTION
**Pull request by brian-luoma**

### CODE REVIEW

File: public/blog/blog_before.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ img[alt='tigers']
...
 <!--
-Currently we are in **summer mode** and we will only be meeting once a month. weekly meetings will begine in September.
+Currently we are in **summer mode** and we will only be meeting once a month. weekly meetings will begin in September.
 -->
 
 </article>
 
 ## Catonsville Cub Scout Pack 306
 
-We meet Wednesday Nights 7-8 pm at Catonsville Presbyterian Church between the first Wednesday of September and the last Wednesday of May. Currently we are meeting on line only but stay tuned for changes as we battle through this virus.
```
Comment (jceaser) [inline public/blog/blog_before.md:25]: wow really, this was still up. Good to get rid of this

File: public/index.md [hunk 2]
```diff
@@ -17,7 +19,7 @@ What is cub scouts and what do we do? Games, activities, crafts, STEM, leadershi
 * [Wolf](dens/wolf/) - second grade
 * [Bear](dens/bear/) - third grade
 * [Webelos 1](dens/webelos1) - fourth grade
-* [Webelos 2](dens/webelos2/) (arrow of light cub scouting) - fifth grade
+* [AArrow of Light](dens/webelos2/) - fifth grade
```
Comment (jceaser) [inline public/index.md:22]: two a's here, can approve, but this should be fixed

**Review by jceaser — APPROVED** [review]
You may want to go back and fix that one file with AArrow

**Comment by brian-luoma** [comment]
Can you delete the additional "A". If not I will do it tonight and send the
new pull request.

**Pull request merged**